crates/libtiny_client/src/lib.rs

```diff
@@ -63,11 +63,21 @@ pub struct ServerInfo {
     pub sasl_auth: Option<SASLAuth>,
 }
 
-/// SASL authentication credentials
+/// SASL authentication mechanisms
+/// https://ircv3.net/docs/sasl-mechs
+/// https://www.alphachat.net/sasl.xhtml
```

Comment:
```suggestion
/// SASL authentication mechanisms
/// - <https://ircv3.net/docs/sasl-mechs>
/// - <https://www.alphachat.net/sasl.xhtml>
```

Just to make it render nicely.